superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -710,14 +710,17 @@ class SqlEditor extends React.PureComponent {
         ? 'Specify name to CREATE VIEW AS schema in: public'
         : 'Specify name to CREATE TABLE AS schema in: public';
 
+    const leftBarStateClass = this.props.hideLeftBar
+      ? 'schemaPane-exit-done'
+      : 'schemaPane-enter-done';
```

Comment:
Looks like there's a bug in the library: https://github.com/reactjs/react-transition-group/issues/643

Without this, if `this.props.hideLeftBar` was initially set to true the bar wouldn't hide on page load.